### PR TITLE
.NET Framework Samples

### PR DESCRIPTION
**Pull request by ameyer-pivotal**
Adds:
- .NET Framework buildpack example
- .NET Framework stack example
- .NET Framework builder example
- .NET Framework app example

### CODE REVIEW

**Comment by jromero** [comment]
100+ files for a sample app? Is there anyway to minimize the number of files?

**Comment by ameyer-pivotal** [comment]
> 100+ files for a sample app? Is there anyway to minimize the number of files?

I suppose it is quite large. We were essentially trying to use a published sample that MS uses (based on this: https://github.com/microsoft/dotnet-framework-docker/tree/master/samples/aspnetapp) to show something that's more like a "real" app. That said, I agree this is large and will investigate alternatives

**Comment by ameyer-pivotal** [comment]
@jromero The sample app should be much simpler now (thanks @micahyoung!)

**Comment by ameyer-pivotal** [comment]
@jromero Hold off on merging this, as I want to put some logos back into this sample app

**Comment by ameyer-pivotal** [comment]
> @jromero Hold off on merging this, as I want to put some logos back into this sample app

@jromero OK, all good now

**Comment by ameyer-pivotal** [comment]
@jromero Heads up, the checks failed to report back, but they're all green from what I can tell

File: buildpacks/dotnet-framework/bin/detect.bat [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@echo off
+
+set plan_path=%2
+
+:: TODO: Should this require/provide something meaningful?
```
Comment (jromero) [inline buildpacks/dotnet-framework/bin/detect.bat:5]: If there is no intention to answer this question as part of this PR then I would ask that we remove this TODO.

Comment (ameyer-pivotal) [reply buildpacks/dotnet-framework/bin/detect.bat:5]: Ah thanks for catching this. Just for completeness, I'll address this

Comment (ameyer-pivotal) [reply buildpacks/dotnet-framework/bin/detect.bat]: Done.

**Pull request merged**